src/main/java/com/github/mathematicalguy/MoeCraftMod.java

```diff
@@ -1,7 +1,14 @@
 package com.github.mathematicalguy;
```

Comment:
We should add another level to the package - typically java is "domain" + "application" - so in this case we should refactor our root package to be `com.github.mathematicalguy.moecraft`

Reply:
@mathematicalguy  I created Issue #5 for this, when you've figured out how to refactor this, you should close the issue.